make_test_images/Cargo.toml

```diff
@@ -11,6 +11,7 @@ anyhow = "1.0.40"
 c2pa = { path = "../sdk", default-features = false, features = [
 	"openssl",
 	"unstable_api",
+	"file_io",
```

Comment:
why do we need to add file_io here? I'd updated this to not require it.

Reply:
I got build errors without it.